### PR TITLE
update headers and copyrights to correct wording

### PR DESCRIPTION
**Pull request by myronkscott**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -99,21 +113,12 @@
       </plugin>
     </plugins>
   </build>
-
-  <distributionManagement>
```
Comment (chrisdennis) [inline pom.xml:103]: Is it safe to remove the distributionManagement section? Won't that break the releaser jobs?

**Review by chrisdennis — CHANGES_REQUESTED** [review]

**Comment by cljohnso** [comment]
The `azure-pipelines.yml` and `Jenkinsfile` files have old copyright statements.

**Review by cljohnso — CHANGES_REQUESTED** [review]
`azure-pipelines.yml` and `Jenkinsfile` files need addressing; the rest looks ok to me.

File: pom.xml (hunk 1 above)
Comment (myronkscott) [reply pom.xml:103]: core does not have these sections which gave me some confidence it was historical artifact.  I intended to add back if necessary.

**Review by cljohnso — APPROVED** [review]

**Review by chrisdennis — APPROVED** [review]

**Pull request merged**